rust/lib/src/lib.rs

```diff
@@ -185,8 +193,29 @@ pub fn save_to_b64() -> String {
     }
 }
 
+fn set_server(server: String) -> String {
+    if let Some(server) = server.try_into().ok() {
+        if let Ok(mut lc) = LIGHTCLIENT.lock() {
```

Comment:
@juanky201271 is seeing some interesting errors...  I am unclear on whether he's testing _this_ feature/code..  but the errors involve locking, and they're new..  so I am guessing it has to do with this.